plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -154,8 +151,18 @@ private function has_existing_tax_rates() {
 	 *
 	 * @return void
 	 */
-	public function track_actioned_on_tax_rate_added() {
+	public function on_tax_rate_added() {
 		$this->mark_actioned();
+		wp_cache_delete( self::TAX_RATE_EXISTS_CACHE_KEY );
```

Comment:
Could we go ahead and set the cache to `true` here to save ourselves a query on the next task load?

```suggestion
		wp_cache_set( self::TAX_RATE_EXISTS_CACHE_KEY, true );
```